iphone/Maps/UI/Settings/MWMTTSLanguageViewController.swift

```diff
@@ -0,0 +1,73 @@
+import AVFoundation
+
+class MWMTTSLanguageViewController: MWMTableViewController {
```

Comment:
@biodranik should we hide MWM prefix in swift? Like that:
```swift
@objc(MWMToast)
final class Toast: NSObject {

```

Reply:
@kirylkaveryn what is the benefit of using a prefix in ObjC code? Is it for distinguishing it from Swift code and classes?

1. The prefix definitely should be hidden or absent in Swift.
2. Can we avoid using it at all in ObjC without any cons?

Reply:
It's maps me style (MapsWithMe). If there are no benefits, and there are no clashes between similar Swift and ObjC files/classes, it's better to remove the prefix.

I assume that it was added to distinguish Swift from ObjC, it may make sense to check git and Github omim's history.

If using prefixes is needed, OM prefix would be better than MWM. But just changing old prefixes to new ones may not be the best idea.